### ISSUE TITLE
Language addition request

### ISSUE THREAD

**Issue by milovincent** [mame/quine-relay#87]
Could you possibly add Malbolge to this thing? Or is it too daunting?

Before you ask, I ain't gonna do it.

**Comment by mame** [comment]
Good point!  Ubuntu has a package of Malbolge, so it satisfies the [Criteria for language inclusion](https://github.com/mame/quine-relay/wiki/Criteria-for-language-inclusion).  Actually I have considered if I can include it.

However, it is very difficult.  One Malbolge program can contain only 22143.375 bytes for the entropy.  (The maximal length of a Malbolge program is 3^10 = 59,049 bytes and every cell can only be initialized with one of 8 characters, so it can have `log(8**(3**10), 256)`.)   The next of Malbolge is Maxima whose size is 23,075 (> 22143.375) bytes.  So we need to compress the Maxima program first...

PR is welcome! :smile:

**Comment by esoteric-programmer** [comment]
What about Malbolge Unshackled? In contrast to Malbolge, it has no memory limit. I guess there is no ubuntu package, but a small interpreter in Haskell (public domain).
It may be really hard to add since printing a string is highly non-trivial in Malbolge Unshackled.

**Comment by mame** [comment]
Oh, I've never heard of that.  Therefore, it does not the [Criteria for language inclusion](https://github.com/mame/quine-relay/wiki/Criteria-for-language-inclusion): "It is a *famous* esoteric language..." "Whether it is famous or not, is determined from *my* point of view." :smirk:  Thank you anyway for letting me know.

If [Hisashi Iizawa's method](https://www.trs.css.i.nagoya-u.ac.jp/person/nishida/DB/pdf/iizawa05ss2005-22.pdf) is still applicable to Malbolge Unshackled, I guess it is (theoretically) possible to include it.  Seeing the order of the languages, Lua could be used to do the encoding.

**Comment by SidBlack** [comment]
Udostępnij swój kod 👍
